src/Resources/Resources/ChangeLog.md

```diff
@@ -18,6 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Add support for new api version 2019-05-10 for `New-AzResourceGroupDeployment`
```

Comment:
nit: I think the API version change is for all cmdlets.

Reply:
Truely. I have updated it. Thanks.